internal/kafka/handler_test.go

```diff
@@ -81,6 +81,8 @@ var _ = Describe("Kafka message handler", func() {
 		})
 
 		It("Should not insert with negative retries", func() {
+			// This is a nasty gloabl unfortunately
+			originalDBRetries := cfg.DatabaseConfig.DBRetries
 			cfg.DatabaseConfig.DBRetries = -1
```

Comment:
Can you try making a copy of the config struct and then setting `DBRetries` to -1 in the copy and passing the copy of the config struct to the `onMessage()` method?

By using a copy, you will not have to record/reset the original setting.

Reply:
Bah good point, changed!